src/components/articles/SearchResults.scss

```diff
@@ -0,0 +1,15 @@
+.search-results {
+  margin-left: auto;
+  margin-right: auto;
+  width: 65%;
+}
+
+.icon-text {
+  margin-right: 8px;
+}
+
+.search-result-image {
```

Comment:
with `scss` you can nest your styling to something like:

```
.search-results {
  margin-left: auto;
  margin-right: auto;
  width: 65%;

 $-image {
      height: 150px;
      object-fit: contain;
      width: 250px;
  }
}
```

Reply:
Working on it.

Reply:
Done